aclk/mqtt.c

```diff
@@ -320,16 +330,27 @@ int _link_subscribe(char *topic, int qos)
 int _link_send_message(char *topic, unsigned char *message, int *mid)
 {
     int rc;
+    size_t write_q, write_q_bytes, read_q;
 
     rc = mosquitto_pub_topic_check(topic);
 
     if (unlikely(rc != MOSQ_ERR_SUCCESS))
         return rc;
 
     int msg_len = strlen((char*)message);
-    info("Sending MQTT len=%d starts %02x %02x %02x", msg_len, message[0], message[1], message[2]);
+    lws_wss_check_queues(&write_q, &write_q_bytes, &read_q);
     rc = mosquitto_publish(mosq, mid, topic, msg_len, message, ACLK_QOS, 0);
 
+#ifdef NETDATA_INTERNAL_CHECKS
+    char msgHead[64];
+    memset(msgHead,0,sizeof(msgHead));
```

Comment:
I know it is not the highest priority for this PR, so we need to have another PR to put spaces between variables in this new block.

Reply:
Already done it ;)